tdrs-frontend/src/components/Profile/Profile.jsx

```diff
@@ -32,23 +32,26 @@ function Profile() {
         </div>
       )}
       <div>
-        <p className="text-bold">
+        <p id="full-name" className="text-bold">
           {user?.first_name} {user?.last_name}
         </p>
         <p>{user?.email}</p>
         <p>{primaryRole?.name}</p>
+        {(() => {
```

Comment:
Might be cleaner and easier to visually parse to have this arrow function as its own function. I'd defer to someone else on that though.